examples/1-with-urql-client/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "next-urql": ["../../"]
+    },
+    "target": "es5",
```

Comment:
Good call just targeting `es5` so we don't have to do any additional Babel transpilation ✅